styles/banner.css

```diff
@@ -0,0 +1,40 @@
+.Cart {
+  font-family: sans-serif;
+}
+
+p {
+  margin: 0;
```

Comment:
This one will improve when we use styled-components. But generally speaking, this is one of the problems with plain CSS as this rule will apply to each and every paragraph in the whole application.